pkg/server/service.go

```diff
@@ -179,10 +180,7 @@ func (c *criService) Run() error {
 
 	// Start event handler.
 	logrus.Info("Start event monitor")
-	eventMonitorCloseCh, err := c.eventMonitor.start()
-	if err != nil {
-		return errors.Wrap(err, "failed to start event monitor")
-	}
+	eventMonitorErrCh := c.eventMonitor.start()
```

Comment:
you can still check if eventMonitorErrCh == nil and leave the old return message if it does, though the make chan error should always work...

Reply:
Yeah, don't think we need to check. :) If it is nil, it is our problem.